docs/advanced-usage.md

```diff
@@ -12,6 +12,16 @@ If for some reason, you don't want the plugin to output `<title>` tags on each p
 ```
 <!-- {% endraw %} -->
 
+### Disabling `generator tag and comments` output
+
+If for some reason, you don't want the plugin to output `<!-- Begin Jekyll SEO tag v2.5.0 --> <meta name="generator" content="Jekyll vx.x.x" /><!-- End Jekyll SEO tag -->` tags on each page, simply invoke the plugin within your template like so:
```

Comment:
> If for some reason

Per @ashmaroli in https://github.com/jekyll/jekyll-seo-tag/issues/307#issuecomment-428299857, I believe the maintainers' consensus was that this feature should not be implemented within the project absent a compelling use case to justify the necessary added complexity.